app/src/main/java/com/android/sample/ui/activitydetails/ActivityDetailsScreen.kt

```diff
@@ -131,6 +134,22 @@ fun ActivityDetailsScreen(
             Spacer(modifier = Modifier.width(4.dp))
             Text(text = dueDate ?: "not defined yet")
           }
+/*
```

Comment:
maybe this part could be merged in a single row or keep it this way and use different icons or else the UI will not look good.